### DISCUSSION TITLE
Gauge panel support for background color based on tresholds

### DISCUSSION THREAD

**Discussion by clompsy** [grafana/grafana#38065] [Dashboards, Panels & Visualizations]
### What
Please add the option to use treshold colors for the background of Gauge panels.

### Background
With Singlestat it was possible to color the background of the panel based on the treshold colors, just like it currently is with "Stat" panels (optionally using the "Color Mode: None/Value/Background" parameter set to "Background").

This option is missing for the Gauge panel entirely making it difficult to see the status from further away if the panel is shown in a small size.

Please add the option to color the background based on the treshold just like the "Stat" panel does.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
